specification/dimensions/pricingcategory.md

```diff
@@ -0,0 +1,47 @@
+# Pricing Category
+
+Pricing Category describes the kind of pricing model used for a charge at the time it was used or purchased. Pricing Category is commonly used for scenarios like differentiating between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+PricingCategory MUST follow the requirements listed below:
+
+* PricingCategory MUST be present in the billing data and MUST be of type String.
+* PricingCategory MUST be null if SkuPriceId is null and MUST NOT be null if SkuPriceId is not null.
+* PricingCategory MUST be one of the allowed values.
+* PricingCategory MUST be "On-Demand" when pricing is predetermined at the standard rate for the billing account.
+* PricingCategory MUST be "Commitment-Based" when CommitmentDiscountId is not null.
+* PricingCategory MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing.
+* PricingCategory MUST be "Other" when there is a pricing model but none of the current allowed values apply.
+
+## Column ID
+
+PricingCategory
+
+## Display name
+
+Pricing Category
+
+## Description
+
+Describes the kind of pricing model used for a charge at the time it was used or purchased.
+
+## Content constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values:
+
+| Value            | Description                                                                                                                                                                                                  |
+| :--------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| On-Demand        | Charges priced at the standard rate for the billing account. This includes any applicable negotiated discounts and volume/tiered pricing, but does not include dynamic or commitment-based discount pricing. |
```

Comment:
I think On-demand isn't a good value here if I'm holistically looking to define this highest-level categorization. I say that knowing the term on-demand has been used to describe a few different things by cloud providers including the pricing model. The issue with on-demand is, as Shawn called out in a previous iteration, it means you're able to purchase something without purchasing discounts or committing to something. Any regular Joe/Jane can walk up and get this price with no commitments. To me, dynamic priced offerings can be described the same way. While they offer reduced rates, a consumer is getting the same exact pricing whenever the SKU is made avaialble at the published price - It's consumed in an on-demand manner.

For me, Fixed or Standard pricing seems like a better option here. If we introduce Fixed|Standard here, Tiered and Dynamic also naturally fit in as a pricing category. On-demand and commitment-based move to the purchase-model (or pricing subcategory in post v1.0).

Given it's late, I'm ok for us to push forward with this for 1.0 preview but curious to see what others think. 


| Description                                                                    | Pricing Model / Pricing Category | Purchasing Model / Pricing Subcategory |
|--------------------------------------------------------------------------------|----------------------------------|----------------------------------------|
| Purchase something as a one-time buy (cert, domain)                            | Fixed                            | On-demand                              |
| Purchase something on a recurring basis (RI with partial upfront)              | Fixed                            | On-demand                              |
| Use something based on on-demand prices with no commitments (EBS volume usage) | Fixed                            | On-demand                              |
| Use something based on tiered prices with no commitments (S3 storage)          | Tiered                           | On-demand                              |
| Use something based on a commitment (EC2 with RI commitment)                   | Discounted                       | Commitment-based                       |
| Unused commitment for a RI                                                     | Discounted                       | Commitment-based                       |
| Use something based on a commitment (EC2 with RI commitment) negation          | Discounted                       | Commitment-based                       |
| Use a Spot instance                                                            | Dynamic                          | Spot                                   |
| I Purchase something that is part of a bundle                                  | Fixed                            | Bundled                                |
| I use a service offered as preview pricing                                     | Fixed                            | Preview                                |
| Use a Interuptable VM (OCI)                                                    | Fixed                            | Interuptable                           |

Reply:
I think directionally, the feedback on "levels" of categorization make sense for post v1.0 enhancements. Current looks good to go. 

Based on the example, sounds like "Fixed" would be constrained to charges where `SkuId` does not have many  `SkuPriceIds`, in order for `Tiered`, `Discounted`, `Dynamic` to make sense as peer high-level categorizations. 

I don't think we currently have any collisions, but could there be a scenario in the future where providers include `Tiered` priced offerings as eligible for 'Discounted' mechanisms as well? Particularly with pricing models like GCP Bigquery (editions, capacity vs on-demand, omni, etc.) we will likely need to be prescriptive with how the data is mapped to columns. 

In a way `PricingCategory` is also describing publicly available opportunities for rate-optimization. 

#### Local SSD charge pricingCategory based on its VM
Fixed: Standard VM
Dynamic: Spot VM
Tiered: n/a
Discounted: 1 or 3 yr CUD with local SSD reservation

Reply:
We've discussed this a few times and keep coming back to On-Demand being the clearest, most obvious term that practitioners are already familiar with.
```suggestion
| On-Demand        | Charges priced at the standard rate for the billing account. This includes any flat rate and volume/tiered pricing but does not include dynamic or commitment-based discount pricing. |
```